js/app/modules/selection/search.js

```diff
@@ -26,7 +26,6 @@ const Search = new Module.Class({
         return Eknc.QueryObject.new_from_props({
             query: query,
             limit: limit,
-            tags_match_all: ['EknArticleObject'],
```

Comment:
Even though I am no longer restricting the search to only Articles, I still do not get video content returned in search results. i think maybe video titles aren't getting indexed? @ffarfan